README.md

```diff
@@ -67,6 +67,19 @@ Slime.render(source, site_title: "Website Title")
 
 ## Reference
 
+### Tags
+
+Starting a line with a string followed by a space will create an html tag, as follows:
+
+```slim
+tt
+  Always bring a towel.
+```
```

Comment:
````suggestion
```slim
tt Always bring a towel.
```
````

If you split the tag across multiple lines you have to use `|`.